data/theleocodes.json

```diff
@@ -0,0 +1,29 @@
+{
+    "name": "Tanvi Saini",
+    "type": "personal",
+    "displayStatsPublic": true,
+    "bio": "**MERN-Stack Developer | Cloud-Computing",
+    "tags": ["ReactJS", "JavaScript", "Cloud", "MongoDB", "NodeJS"],
+    "socials": [
+      { "icon": "FaTwitter", "url": "https://twitter.com/" },
+      { "icon": "FaGithub", "url": "https://github.com/theleocodes" }
+    ],
+    "links": [
+        {
+          "name": "Follow me on GitHub",
+          "url": "https://github.com/adityassharma-ss",
```

Comment:
you'll have to update the github username in the url

Reply:
Good spot @ChinmayMhatre 👍